app/assets/javascripts/utils/article_finder_utils.js

```diff
@@ -0,0 +1,82 @@
+import logErrorMessage from './log_error_message';
+
+const pageviewBaseUrl = 'https://wikimedia.org/api/rest_v1/metrics/pageviews/per-article/en.wikipedia/all-access/user/';
```

Comment:
This should not be en.wikipedia specific. We might roll this out for just en.wikipedia to start with, but we should write to only keep the specific wiki in one place (like the top ArticleFinder component) so that it''ll be easy to adapt for other langauges.